unit-tests/resources/commcare-apps/case_search_and_claim/default/app_strings.txt

```diff
@@ -0,0 +1,45 @@
+app.display.name=village-health2
```

Comment:
I think you can remove this file; I think it's good for us to keep the test app files as bare-bones as possible